### ISSUE TITLE
Automate signing key generation

### ISSUE THREAD

**Issue by bbockelm** [opensciencegrid/SciTokens-CredMon#4]
If `scitokens-credmon` is running as root, it should;
1.  See if the signing key already exists.
2.  If not, generate a new one and drop it into place.
3.  Look at the configured issuer and download the list of known keys.
4.  If the current credential is not in the list of known keys, it should Complain Loudly and log a copy/paste-able JSON snippet (including key ID), asking the sysadmin to provide the (possibly freshly generated) key to the issuer.
5.  If the current credential is in the list of known keys, check the local key ID and make sure it is set appropriately.

Basically, we want to reduce and simplify the steps for registering a new schedd within the issuer infrastructure.
